packages/web-components/fast-components/src/patterns/start-end.ts

```diff
@@ -28,7 +28,7 @@ export const endTemplate = html<StartEnd>`
 `;
 
 export const startTemplate = html<StartEnd>`
-    <span name="start" part="start" ${ref("startContainer")}>
+    <span class="start" name="start" part="start" ${ref("startContainer")}>
```

Comment:
Same as above.

Reply:
Yup I think your right, `name` is not necessary here, I noticed that but was reluctant to remove it. I'll update